base/compiler/ssair/ir.jl

```diff
@@ -637,7 +636,7 @@ mutable struct IncrementalCompact
         return new(code, parent.result,
             parent.result_bbs, ssa_rename, bb_rename, bb_rename, parent.used_ssas,
             late_fixup, perm, 1,
-            new_new_nodes, new_new_used_ssas, pending_nodes, pending_perm,
+            new_new_nodes, parent.new_new_used_ssas, pending_nodes, pending_perm,
```

Comment:
I'm not sure this makes sense. If this has its own `new_new_nodes`, then the ssa count should go with it.

Reply:
That said, I'm not sure the new_new_nodes facility is used by inlining at all.

Reply:
I was just mirroring how `used_ssas` is used by this constructor, so that calls to stuff like `setindex!`, `insert_node_here!` and whatnot during inlining update the parents `new_new_used_ssas` like how they update `used_ssas`. Otherwise, changes made there will just be in the temporary `inline_compact` object's fields

Reply:
Oops, yeah, didn't realize this has its own `new_new_nodes`, though...